cros_gralloc/cros_gralloc_driver.cc

```diff
@@ -454,7 +481,6 @@ int32_t cros_gralloc_driver::allocate(const struct cros_gralloc_buffer_descripto
 		hnd->fds[i] = -1;
 
 	hnd->num_planes = num_planes;
-	hnd->from_kms = false; // not used, just set a default value. keep this member to be backward compatible.
```

Comment:
from_kms is handle specific. not driver specific. should not make it as driver structue's member

Reply:
ok